frontend/src/main/web/user-profile-page/README.md

```diff
@@ -0,0 +1,15 @@
+# React Profile
+
+## Setup
+
+`npm install`
+
+## Options
+
+### Production Build
+
+`npm run build`
+
+### Development Server
```

Comment:
Just a http server to server up js,html and css files. Some things won't work if you access the browser not using http (i.e. file://some/path/to/your/local/file)